src/components/buttonCreator/ButtonCreator.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import "src/components/buttonCreator/ButtonCreator.component.scss";
+
+export const ButtonCreator = ({
+  onButtonClick,
+  buttonText,
+  icon,
+  altText,
+  buttonClass,
+}) => {
+  return (
+    <button className={"button " + buttonClass} onClick={onButtonClick}>
+      <img src={icon} alt={altText} /> {buttonText}
```

Comment:
src can also be represent as scrIcon like you did in alt
```suggestion
      <img src={srcIcon} alt={altText} /> {buttonText}
```